world/dominion/plots/models.py

```diff
@@ -806,7 +806,7 @@ def check_org_busy(self: "PlotAction"):
             if (
                 self.org.actions.exclude(status=PlotAction.DRAFT)
                 .exclude(status=PlotAction.CANCELLED)
-                .filter(Q(beat__isnull=True) | Q(beat__episode=episode))
+                .filter(Q(episode=episode))
```

Comment:
Q() with a single arg is a no-op, so you can remove that. It can just be filter(episode=episde)

Reply:
Fixed